Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/KeyboardTypeModifier.swift

```diff
@@ -20,7 +20,7 @@ import SwiftUI
 #if swift(>=5.8)
 @_documentation(visibility: public)
 #endif
-@available(iOS 16.0, tvOS 16.0, *)
```

Comment:
why is tvos being removed?

Reply:
@bcardarella I'm trying to figure out why CI is failing.

Reply:
ah ok :)